lighthouse-core/test/audits/accessibility/aria-allowed-attr-test.js

```diff
@@ -5,8 +5,8 @@
  */
 'use strict';
 
-const Audit = require('../../../audits/accessibility/aria-allowed-attr.js');
-const assert = require('assert').strict;
+import Audit from '../../../audits/accessibility/aria-allowed-attr.js';
+import {strict as assert} from 'assert';
```

Comment:
nit: do we want to start doing `import assert from 'assert/strict';`? Seems way better to me

Reply:
We use this current form in other places to, so let's handle it everywhere after this PR lands. tracking #13883